apps/android/package.json

```diff
@@ -44,7 +44,7 @@
     ],
     "entry": "index.js",
     "outputPath": "./dist",
-    "bundleName": "index.bundle",
+    "bundleName": "index.android.bundle",
     "noPlatformSuffix": true
```

Comment:
I think our tooling is set up so that you can set `noPlatformSuffix` to false if you want to include the platform in the bundle.

Reply:
it adds the suffix at last like index.bundle.android, so falling back to original naming convention